low-level/files/opt/seagate/sspl/setup/generate_resource_health_view/resource_health_view

```diff
@@ -797,38 +753,6 @@ class SSPLHealthView(object):
                 store.put(system[singleton_realstorencl.FAULT_KEY],
                     singleton_realstorencl.faults_persistent_cache)
 
-    def build_encl_hw_specifics_instances(self):
```

Comment:
Why did we move this to manifest_support_bundle?

Reply:
Earlier, there was a consul dependency because of which manifest support bundle was internally calling resource health view file.
Now we have removed that dependency and manifest support bundle can execute independently.